frontend/src/views/Game.tsx

```diff
@@ -10,20 +16,45 @@ type LocationState = {
 function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
+  const [problems, setProblems] = useState<Problem[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
 
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((res) => {
+      if (isError(res)) {
+        setError((res as ErrorResponse).message);
+        setProblems(null);
+      } else {
+        setProblems(res as Problem[]);
+        setError(null);
```

Comment:
Same here; not needed I believe.

Reply:
Same response as above

Reply:
Same as my previous comment about whether this has been changed to `setError('')`. I may also just be looking at an outdated version so sorry if that's the case